terraform/modules/monitoring/policy.tf

```diff
@@ -0,0 +1,30 @@
+resource "aws_sns_topic_policy" "this" {
+  arn = aws_sns_topic.this.arn
+
+  policy = data.aws_iam_policy_document.this.json
+}
+
+data "aws_iam_policy_document" "this" {
+  statement {
+    actions = [
+      "SNS:Publish"
+    ]
+
+    effect = "Allow"
+
+    resources = [
+      aws_sns_topic.this.arn
+    ]
+
+    principals {
+      type        = "AWS"
+      identifiers = ["*"]
+    }
+
+    condition {
+      test     = "ArnLike"
+      variable = "aws:SourceArn"
+      values   = []
```

Comment:
Currently nothing is allowed to publish - my plan was to add ARNs here for resources as they are created.

Reply:
Looks good (presuming it's the arn's of the `aws_cloudwatch_metric_alarm`'s that will be added here)